internal/pkg/template/templates/environment/cf.yml

```diff
@@ -397,8 +420,7 @@ Outputs:
     Export:
       Name: !Sub ${AWS::StackName}-SubDomain
   EnabledFeatures:
-    # We don't need to include Aliases because updating it always results in the CustomDomain action to update.
-    Value: !Sub '${ALBWorkloads},${EFSWorkloads},${NATWorkloads}'
+    Value: !Sub '${ALBWorkloads},${EFSWorkloads},${NATWorkloads},${Aliases}'
```

Comment:
dumb q: why do we now want to include Aliases?

Reply:
Explained here and i'll take another routehttps://github.com/aws/copilot-cli/pull/3386#discussion_r844515209